cat/CaT-service.yaml

```diff
@@ -11,6 +11,181 @@ servers:
+      responses:
+        '200':
+          description: Success for the update.
+          content:
+            application/json:
+              schema:
+                $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-conclave-api-definitions/main/UserProfile/UserService.yaml#/components/schemas/UserContactInfoList'
+        '400':
+          $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OpenAPI_Standards/CCS_Standard_API_Responses.yaml#/components/responses/400InvalidRequest'
+        '401':
+          $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OpenAPI_Standards/CCS_Standard_API_Responses.yaml#/components/responses/401Unauthorized'
+        '403':
+          $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OpenAPI_Standards/CCS_Standard_API_Responses.yaml#/components/responses/403Forbidden'
+        '404':
+          $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OpenAPI_Standards/CCS_Standard_API_Responses.yaml#/components/responses/404NotFound'
+        '429':
+          $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OpenAPI_Standards/CCS_Standard_API_Responses.yaml#/components/responses/429TooManyRequests'
+        '500':
+          $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OpenAPI_Standards/CCS_Standard_API_Responses.yaml#/components/responses/500InternalServerError'
+        '502':
+          $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OpenAPI_Standards/CCS_Standard_API_Responses.yaml#/components/responses/502BadGateway'
+        '504':
+          $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OpenAPI_Standards/CCS_Standard_API_Responses.yaml#/components/responses/504GatewayTimeout'
+        default:
+          description: successful operation
+      summary: Update user details in Tenders (Jaggaer)
+      
+  /tenders/orgs/{org-id}:
+    parameters:
+      - $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OpenAPI_Standards/CCS_Standard_API_Params.yaml#/components/parameters/OrgParam'
+    patch:
+      tags:
+        - Profile Management
+      description: Updates an existing Tenders API Org with Conclave organisation details (Conclave access only).
+      operationId: app.updateOrg
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-conclave-api-definitions/main/OrgProfile/OrgService.yaml#/components/schemas/OrganisationProfileInfo'
```

Comment:
Just to clarify - for the user creation, the endpoint `PUT /tenders/users/{user-id}` has no request body and therefore implies we must look up the user's record in CII - but here we are receiving all the data we need to create the org? Or have I misunderstood..

Reply:
The idea of the PUT is that the service calling it does not have access to the user data. It is held in Conclave User Profile. The users profile will reference and organisation held in the Organisation Profile. The PATCHs are pub/sub on the org profile and user profile data. I need to discuss with you if it is better to have separate end points or use the pub/sub just as a trigger for the tenders API to get the updates itself.

Reply:
@tommyb82 I ave also thought about this a bit more. The PATCH is likely to be authenticated as a system i.e. the Adapter Service where as the PUT is going to be authenticated as a user i.e. the user should be the owner of the user-id or an admin user. Currently the user-id must always be that of the requesting user.